src/pocketmine/block/Block.php

```diff
@@ -641,7 +641,7 @@ public function getBoundingBox(){
 	/**
 	 * @return AxisAlignedBB
 	 */
-	protected function recalculateBoundingBox(){
+	protected function recalculateBoundingBox() : AxisAlignedBB{
```

Comment:
In my opinion, it is better to start the PHP 7 updates after 7.1 so that we can add everything together, reducing the risk of inconsistency (even if we don't really care about the consistency between different non-master commits).